### ISSUE TITLE
err gives invalid commands back

### ISSUE THREAD

**Issue by Eagllus** [gbin/err#530]
I reverted my err version to 3.0.4
I get the following message
```
Version 3.1.1 of err is available. http://pypi.python.org/pypi/err/3.1.1. You can disable this check by doing !unload VersionChecker followed by !blacklist VersionChecker
```

now either !unload and !blacklist are not valid commands.
They have been changed to `!plugin deactivate` and `!plugin blacklist`

also everytime I now install a plugin this is the response I'm getting back
```
A new plugin repository has been installed correctly from https://github.com/daenney/err-githubhook.git. Refreshing the plugins commands...
08:58
Notice: VersionChecker is blacklisted, use !load VersionChecker to unblacklist it
08:58
Notice: VersionChecker is blacklisted, use !load VersionChecker to unblacklist it
```

**Comment by gbin** [comment]
docs/user_guide/plugin_development/basics.rst is outdated too.

**Comment by gbin** [comment]
Fixed by 23fa73055051312312bcb8005949c401d29f7f5e
